types/database/database.d.ts

```diff
@@ -697,7 +697,7 @@ export type LoginArgs = {
   password: string;
 };
 
-export type LoadMeArgs = { user: RawUser; deviceToken?: string };
+export type LoadMeArgs = { user?: RawUser; deviceToken?: string };
```

Comment:
why would user be undefined?

Reply:
On the SSO Login flow, we retrieve the deviceToken only and then use it to call the `loadMe` request without passing the user profile. Only in there, that we retrieve the user profile via the `Client4.getMe()`.